finders/schemas/utiac-decisions.json

```diff
@@ -0,0 +1,518 @@
+{
+  "document_noun": "decision",
+  "facets": [
+    {
+      "key": "country",
+      "name": "Country",
+      "type": "text",
+      "preposition": "related to",
+      "display_as_result_metadata": true,
+      "filterable": true,
+      "allowed_values": [
+        {"label": "No Country", "value": "No Country"},
```

Comment:
Thanks for clarifying the convention for `value` formatting. I'll convert all to the parameterised version.